client/my-sites/earn/home.tsx

```diff
@@ -212,6 +213,65 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 		};
 	};
 
+	/**
+	 * Return the content to display in the Donations card based on the current plan.
+	 *
+	 * @returns {object} Object with props to render a PromoCard.
+	 */
+	const getDonationsCard = () => {
+		const cta = isFreePlan
+			? {
+					text: translate( 'Unlock this feature' ),
+					action: () => {
+						trackUpgrade( 'any-paid-plan', 'donations' );
+						page( `/plans/${ selectedSiteSlug }` );
+					},
+			  }
+			: {
+					text: translate( 'Learn more about donations' ),
+					action: () => {
+						trackCtaButton( 'donations' );
+						if ( window && window.location ) {
+							window.location.href = localizeUrl( 'https://en.support.wordpress.com/donations/' );
+						}
+					},
+			  };
+		const title = translate( 'Accept donations and tips' );
+
+		const freePlanBody = translate(
+			'Collect donations, tips, and contributions for your creative pursuits, organization, or whatever your website is about. {{em}}Available with any paid plan{{/em}}.',
+			{
+				components: {
+					em: <em />,
+				},
+			}
+		);
+		const paidPlanBody = translate(
+			'Collect donations, tips, and contributions for your creative pursuits, organization, or whatever your website is about.'
+		);
+
+		const body = isFreePlan ? freePlanBody : paidPlanBody;
```

Comment:
May I suggest to look into using the [approach taken by the Pay with PayPal card](https://github.com/Automattic/wp-calypso/blob/31f77bbc42ffe354e5e026ebcfe5fe382d6674b0/client/my-sites/earn/home.tsx#L135-L142) so we can re-use the translation regardless the current plan of the site?

```
const body = (
  <>
    { translate( 'Collect donations, tips, and contributions for your creative pursuits, organization, or whatever your website is about.' ) }
    { isFreePlan && <em>{ translate( 'Available with any paid plan' ) }.</em> }
  </>
);
```